tests/test_warpedvrt.py

```diff
@@ -73,6 +74,49 @@ def test_warped_vrt_dimensions(path_rgb_byte_tif):
         assert vrt.warp_extras == {'init_dest': 'NO_DATA'}
 
 
+def test_warped_vrt_resampling(path_rgb_byte_tif):
+    """
+    Read from a WarpedVRT using different resampling methods.
+    """
+    # window in EPSG 3857 partly overlapping with example TIFF
+    window_bounds = (
+        -8766409.899970274, 2817774.610704731, -8609866.866042234,
+        2974317.6446327716)
+
+    with rasterio.open(path_rgb_byte_tif) as src:
+        # simple read using default nearest interpolation
+        with WarpedVRT(src, dst_crs='EPSG:3857') as vrt:
+            nearest = vrt.read(window=vrt.window(*window_bounds))
+        # initialize VRT with bilinear interpolation
+        with WarpedVRT(
+            src, dst_crs='EPSG:3857', resampling=Resampling.bilinear
+        ) as vrt:
+            bilinear1 = vrt.read(window=vrt.window(*window_bounds))
+        # make sure interpolation was applied
+        assert not np.array_equal(nearest, bilinear1)
```

Comment:
@ungarj I think this test is almost guaranteed to pass because the warp kernel does virtually no resampling between UTM and web mercator for a `WarpedVRT` with the default dimensions. Testing the warp features is challenging, for sure.

Reply:
True, I tried quite a few example windows until the test was ok. It fails however if the ``resampling`` is not passed on.

Should I try with a lat/lon ``WarpedVRT`` instead?